backend/de.metas.util/src/main/java/de/metas/util/async/Debouncer.java

```diff
@@ -133,7 +132,7 @@ private void updateDueTimeAndScheduleTask()
 		final int delayInMillisEffective = bufferMaxSize > 0 && buffer.size() >= bufferMaxSize
 				? 0 // ASAP
 				: delayInMillis;
-		dueTime = SystemTime.millis() + delayInMillisEffective;
+		dueTime = System.currentTimeMillis() + delayInMillisEffective;
```

Comment:
```suggestion
		// we don't use SystemTime because in our usual tests it's rigged to return a fixed value. Fee free to use it here, too - maybe with an enhanced Timesource - when it makes sense
		dueTime = System.currentTimeMillis() + delayInMillisEffective;
```